_posts/2020-04-17-snowplow-ios-tracker-1.3.0-released/index.md

```diff
@@ -0,0 +1,309 @@
+---
+layout: post
+title: "Snowplow iOS Tracker 1.3.0 released"
+title-short: Snowplow iOS Tracker 1.3.0
+tags: [snowplow, objc, ios, tracker]
+author: Alex Benini
+category: Releases
+permalink: /blog/2020/04/17/snowplow-ios-tracker-1.3.0-released/
```

Comment:
will this get updated with today's date?

Reply:
yep, fixing it! 👍 
It's probably something that we should update when we merge on master /cc @nepsil0n